### DISCUSSION TITLE
How to install on Windows with*out* admin privileges?

### DISCUSSION THREAD

**Discussion by ElectricRCAircraftGuy** [BurntSushi/ripgrep#2710] [Q&A] [answered]
Using tools like `choco install ripgrep` requires running inside a privileged admin Git Bash terminal or Command Prompt. What if we don't have admin rights?

**Comment by BurntSushi** [comment]
Can you please post the answer in a response so that I can mark it as an answer? Thank you!

**Comment by ElectricRCAircraftGuy** [comment, marked as answer]
[Here's my answer on this to help others,](https://stackoverflow.com/a/76666895/4561887) for anyone wondering. See Option 1.
